### PR TITLE
Rework base layer form.

### PR DESCRIPTION
**Pull request by badnames**
## Basics

- [x] I added a line to [changelog.md](/doc/changelog.md)
- [x] The PR is rebased with current master.
- [x] Details of what you changed are in commit messages.
- [x] References to issues, e.g. `close #X`, are in the commit messages and changelog.
- [x] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added automated tests or a [manual test protocol](../doc/tests/manual/protocol.md)
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)
- [ ] Exceptions to any guidelines are documented

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] I've read through the whole documentation
- [ ] I've checked conformity to guidelines

### CODE REVIEW

**Comment by markus2330** [comment]
jenkins build please

**Comment by markus2330** [comment]
Thank you for the PR!

- [ ] Can you please change so that the boxes resize according the toolbar (as before)?
- [ ] The tick should be changed to a cross if image cannot be found.

File: frontend/src/config/i18n/de/baseLayerForm.json [hunk 1]
```diff
@@ -4,8 +4,11 @@
   "selectImage": "Hintergrund wählen",
   "rotation_field": "Rotation (Grad)",
   "scale": "Skalierung (pixel/meter)",
+  "scale_invalid": "Die Skalierung muss eine Zahl >= 0 sein!",
```
Comment (absurd-turtle) [inline frontend/src/config/i18n/de/baseLayerForm.json:7]: Consider changing >= to greater or equal than.

**Review by absurd-turtle — APPROVED** [review]
lgtm :)

File: frontend/src/config/i18n/de/baseLayerForm.json (hunk 1 above)
Comment (badnames) [reply frontend/src/config/i18n/de/baseLayerForm.json]: I've now fixed both issues.

**Review by markus2330 — APPROVED** [review]
Works super-nicely! ~~@absurd-turtle can you please review the code?~~

EDIT: Was already done...

**Pull request merged**